src/components/OfferCheckoutCard/OfferCheckoutCard.tsx

```diff
@@ -115,8 +119,83 @@ const OfferCheckoutCard = () => {
     );
   };
 
+  const generateDescriptionForCoupon = () => {
+    const formattedTotalPrice = formatNumber(totalPrice);
+    const regularPrice = calculateGrossPriceForFreeOffer(
+      offerPrice,
+      taxRate,
+      customerPriceInclTax
+    );
+    let description;
+    let formattedDescription;
+    if (discountPeriods === 1) {
+      // non-standard period
+      if (period === '3months' || period === '6months') {
+        formattedDescription = `You will be charged ${currencySymbol}${formattedTotalPrice} (incl. ${taxCopy}) per ${periodMapper[
+          period as Period
+        ]
+          .chargedForEveryText as string} for the next billing period.<br/>After that time you will be charged a regular price of ${currencySymbol}${regularPrice}.`;
```

Comment:
I think this this copy matches the logic of 1 discounted period. User is charged a discounted price now, and all next payments will be a regular price, so for me it should be more like:
You will be charged now ${currencySymbol}${formattedTotalPrice} (incl. ${taxCopy}) **now**. <br/>After **Period (formatted)** you will be charged a regular price of ${currencySymbol}${regularPrice}.`;

Reply:
done